structuredheadings/plugin.js

```diff
@@ -301,22 +317,23 @@
       setupElements(editor);
       setupCommands(editor);
       setupStyles(editor);
+      self.currentScheme = editor.config.autonumberDefaultScheme;
 
       editor.on("dataReady", function () {
         self.currentScheme = self.detectScheme();
       });
 
       //Format Dropdown
       editor.ui.addRichCombo("NumFormats", {
-        label: "Formats",
-        title: "Numbering Formats",
+        label: "Formatting",
+        title: "Text Formatting",
```

Comment:
Are we referring to:

> The tooltip on rollover of the format dropdown says “Numbering Formats”. It should say “Text Formatting”

I don't think that's Jana suggesting that the dropdown be changed to "Text Formatting", just that the tooltip shouldn't say "Numbering Formats" when it's not the numbering section.

I don't see how "Text Formatting" is an improvement over "Formatting".